### PR TITLE
code review from standup team

### PR DESCRIPTION
**Pull request by Mhdez221993**

### CODE REVIEW

**Review by Coolczycki — COMMENTED** [review]
Anchor tags should be nested inside of the list item. Putting them outside of <li> will probably throw out linters error.

File: index.html [hunk 1]
```diff
@@ -85,10 +84,10 @@ <h1>
                   />
                 </figure>
                 <div class="absolute">
-                  <h1>
```
Comment (Coolczycki) [inline index.html]: Is that an h3 tag nested inside of h1 tag

File: index.html [hunk 2]
```diff
@@ -101,10 +100,10 @@ <h1>
...
+                  <h3>
```
Comment (Coolczycki) [inline index.html:103]: Same here

File: index.html [hunk 3]
```diff
@@ -233,10 +232,10 @@ <h1>
             <article class="section-3">
               <header class="red flex-4">
                 <a href="#"><h2>Latest funding rounds on Index.co ></h2></a>
-                  <ul class="flex-4">
-                  <li class="space">Add info</li>
-                  <li class="space">Get updates by email</li>
-                  <li>See more</li>
+                  <ul class="flex-4 red">
+                    <a href="#"><li class="space">Add info</li></a>
```
Comment (Coolczycki) [inline index.html]: Anchor tags should be inside of list item tag

File: index.html [hunk 4]
```diff
@@ -701,7 +701,7 @@ <h2 class="red"> <a href="#">Latest deals > </a></h2>
               </section>
             </div>
     </main>
-    <footer class="footer">
+    <footer id="footer" class="footer">
```
Comment (Coolczycki) [inline index.html:701]: id names should be unique, also you should avoid using the same names as sematic tags for names of your classes and ids

**Review by Coolczycki — COMMENTED** [review]
All the commits should be starting with a capital letter and be written in imperative mode.

**Comment by Mhdez221993** [comment]
Has been fixed

**Pull request closed without merging**